build/teamcity/cockroach/nightlies/pebble_nightly_metamorphic_crossversion.sh

```diff
@@ -22,15 +25,19 @@ mkdir -p $root/artifacts
 
 VERSIONS=""
 LAST_SHA=""
-for branch in "$@"
+for arg in "$@"
 do
-    tc_start_block "Compile Pebble $branch metamorphic test binary"
-    SHA=$("$dir/teamcity/cockroach/nightlies/pebble_nightly_build_test_binary.sh" "$branch" "bin" | tail -n1)
-    VERSIONS="$VERSIONS -version $branch,$SHA,/test-bin/$SHA.test"
+    CRDB_BRANCH="${arg%%:*}"
+    # Extract the part after the colon
+    PEBBLE_BRANCH="${arg#*:}"
+
+    tc_start_block "Compile Pebble $CRDB_BRANCH:$PEBBLE_BRANCH metamorphic test binary"
+    SHA=$("$dir/teamcity/cockroach/nightlies/pebble_nightly_build_test_binary.sh" "$CRDB_BRANCH" "$PEBBLE_BRANCH" bin | tail -n1)
+    VERSIONS="$VERSIONS -version $PEBBLE_BRANCH,$SHA,/test-bin/$SHA.test"
     LAST_SHA="$SHA"
     echo "$PWD/bin/$SHA.test"
     stat "$PWD/bin/$SHA.test"
-    tc_end_block "Compile Pebble $branch metamorphic test binary"
+    tc_start_block "Compile Pebble $CRDB_BRANCH:$PEBBLE_BRANCH metamorphic test binary"
```

Comment:
I assume this was supposed to be `tc_end_block`?

Reply:
Good catch, thanks!